### PR TITLE
Fix small typo in extension hello world tutorial

### PR DESCRIPTION
**Pull request by dstansby**
Subject: minor typo fix

### Feature or Bugfix

- Bugfix

### Purpose
- <long purpose of this pull request>
- <Environment if this PR depends on>

### Detail
- <feature1 or bug1>
- <feature2 or bug2>

### Relates
- <URL or Ticket>

### CODE REVIEW

**Review by jfbu — COMMENTED** [review]
Alas, I don't know (not native) if one should write "is to call" in place of "is call"...

else I would merge this rightaway ;-) and avoid reaching  a disproportionate ratio "length+time of review divided by length+time of PR"...

**Comment by ProGamerGov** [comment]
@jfbu Google docs' autocorrect feature is saying that their change is correct:

it_call

**Comment by fluesvamp** [comment]
> is to call

use the above infinitive form

File: doc/development/tutorials/helloworld.rst [hunk 1]
```diff
@@ -105,7 +105,7 @@ Sphinx.
    :linenos:
    :lines: 12-
 
-The simplest thing you can do it call the :meth:`~Sphinx.add_directive` method,
+The simplest thing you can do is call the :meth:`~Sphinx.add_directive` method,
```
Comment (AA-Turner) [inline doc/development/tutorials/helloworld.rst]:
```suggestion
The simplest thing you can do is to call the :meth:`~Sphinx.add_directive` method,
```

**Review by AA-Turner — CHANGES_REQUESTED** [review]

**Comment by jfbu** [comment]
@AA-Turner I have observed transient errors with linkcheck recently, here it happened on Windows
```
FAILED tests/test_build_linkcheck.py::test_defaults - assert 7 == 6
FAILED tests/test_build_linkcheck.py::test_anchors_ignored - assert not "link...
= 2 failed, 1777 passed, 39 skipped, 5 xfailed, 28 xpassed, 10 warnings in 305.94s (0:05:05) =
ERROR: InvocationError for command 'D:\a\sphinx\sphinx\.tox\py\Scripts\python.EXE' -X dev -X warn_default_encoding -m pytest --durations 25 -vv (exited with code 1)
```
but I have seen it either on py37 or py310 Linux testing too.  Do you know the cause?

Anyway I will merge this because there is good chance testing will pass after merge (based on earlier experiences).

**Review by jfbu — APPROVED** [review]

**Comment by jfbu** [comment]
Thanks @dstansby for contribution, merging!

**Pull request merged**